app/scripts/modules/netflix/fastProperties/domain/acaTaskStage.ts

```diff
@@ -0,0 +1,89 @@
+import {IStage} from '../../../core/domain/IStage';
+import {IUser} from '../../../core/authentication/authentication.service';
+import PropertyCommand from './propertyCommand.model';
+import {Canary} from './canary.domain';
+
+export class AcaTaskStage implements IStage {
+  name: string;
+  type: string;
+  refId: string;
+  requisiteStageRefIds: string[] = [];
+  comments: string;
+  canary: Canary;
+
+
+
+  constructor(user: IUser, command: PropertyCommand, configDetails: any, previousStage?: IStage) {
+    this.refId = previousStage ? `${parseInt(previousStage.refId, 10) + 1}` : '1';
+    this.requisiteStageRefIds = previousStage ? [previousStage.refId] : [];
+    this.type = 'acaTask';
+    this.name = 'ACA Task';
+    this.canary = new Canary(user, command, configDetails);
+    // this.notifications = [{ 'type': 'email', 'address': user.name}];
+  }
+
+// {
+//   "refId": "2",
+//   "requisiteStageRefIds": [
+//     "1"
+//     ],
+//   "type": "acaTask",
+//   "name": "ACA Task",
+//   "baseline": {},
+//   "canary": {
+//     "application": "mahe",
+//     "owner": "zthrash@netflix.com",
+//     "watchers": [
+//       "zthrash@netflix.com"
+//       ],
+//     "canaryConfig": {
+//       "name": "FP - Canary",
+//       "canaryHealthCheckHandler": {
+//         "@class": "com.netflix.spinnaker.mine.CanaryResultHealthCheckHandler",
+//         "minimumCanaryResultScore": "10"
+//       },
+//       "canaryAnalysisConfig": {
+//         "notificationHours": [
+//         1
+//         ],
+//         "useLookback": true,
+//         "lookbackMins": 30,
+//         "useGlobalDataset": true,
+//         "name": "Cadmium_All_Canary_Error_Analysis",
+//         "beginCanaryAnalysisAfterMins": "5",
+//         "canaryAnalysisIntervalMins": "15"
+//       },
+//       "lifetimeHours": "2",
+//       "combinedCanaryResultStrategy": "AGGREGATE",
+//       "canarySuccessCriteria": {
+//         "canaryResultScore": "80"
+//       }
+//     },
+//     "canaryDeployments": [
+//       {
+//         "type": "query",
+//         "@class": ".CanaryTaskDeployment",
+//         "accountName": "prod",
+//         "region": "us-west-1",
+//         "baseline": "baseline query",
+//         "canary": "canary query"
+//       }
+//       ]
+//   },
+//   "continueOnUnhealthy": true,
+//   "sendNotifications": true,
+//   "notifications": [
+//     {
+//       "level": "stage",
+//       "when": [
+//         "stage.failed"
+//         ],
+//       "type": "email",
+//       "address": "zthrash@netflix.com"
+//     }
+//     ],
+//   "comments": "This is the comments for the ACA Fast Property Stage"
+// }
```

Comment:
I put them in to show the structure of the pipeline payload.  I'll take them out.